src/Serialization/LieTheory.jl

```diff
@@ -40,7 +42,7 @@ end
 type_params(e::T) where T <: Union{RootSpaceElem, DualRootSpaceElem} = TypeParams(T, root_system(e))
 
 function save_object(s::SerializerState, r::Union{RootSpaceElem,DualRootSpaceElem})
-  save_object(s, _vec(coefficients(r)))
+  save_object(s, Oscar._vec(coefficients(r)))
```

Comment:
```suggestion
  save_object(s, _vec(coefficients(r)))
```
this gets imported above, so no qualification needed